hc/front/views.py

```diff
@@ -552,3 +552,26 @@ def privacy(request):
 
 def terms(request):
     return render(request, "front/terms.html", {})
+
+
+# Failed Jobs
+@login_required
+def failed_checks(request):
+    q = Check.objects.filter(user=request.team.user).order_by("created")
+    checks = list(q)
+    down_checks = [check for check in checks if check.get_status() == 'down']
+    down_tags = set() 
```

Comment:
Adding to the set here is expensive. We can also leave off the nesting.
`down_tags = set([*check.tag_list() for check in down_checks])`